code/game/gamemodes/changeling/changeling.dm

```diff
@@ -28,6 +29,10 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 
 /datum/game_mode/changeling/pre_setup()
 
+	if(num_players() <= lowpop_amount)
+		if(prob((2*1.14**lowpop_amount)-2)) //exponential equation, chance of restriction goes up as pop goes down.
```

Comment:
You have the thing in the wrong thingy, the way you have it, the more players you have, the less chance you have to have a traitor head